### ISSUE TITLE
Questions about the input for frame synthesis network

### ISSUE THREAD

**Issue by asheroin** [baowenbo/DAIN#34]
As described in figure 3 from the CVPR paper, the input of frame synthesis network consists of five components, including raw interpolation kernels, projected flows, warped depth maps, warped frames and warped context features. However, in line [177](https://github.com/baowenbo/DAIN/blob/master/networks/DAIN_slowmotion.py#L177) to [181](https://github.com/baowenbo/DAIN/blob/master/networks/DAIN_slowmotion.py#L181) from `DIAN_slowmotion.py` , the input for rectifyNet seems  not as same as described:

`rectify_input = torch.cat((cur_output_temp,ref0,ref2, cur_offset_output[0],cur_offset_output[1],  cur_filter_output[0],cur_filter_output[1],  ctx0,ctx2 
                                        ),dim =1)`

It seems that the actual input for the frame synthesis network did not include the warped depth maps while used a blended result from warped frames alternatively. 

So which one should be the correct way for the proposed method?  Would you pleased give a numerical analysis for such different settings?

**Comment by laomao0** [comment]
https://github.com/baowenbo/DAIN/blob/master/networks/DAIN.py
Line 133-138 
The context features include the depth information.
